app/jobs/depositor_email_notification_job.rb

```diff
@@ -8,6 +8,9 @@ def perform
       statistics = user.statistics_for
       next if statistics.nil?
 
+      # Skip if new_work_views or new_file_downloads is 0
+      next if statistics[:new_work_views].zero? || statistics[:new_file_downloads].zero?
```

Comment:
I think this should be an and rather than an or... we'd want to send as long as they have anything non-zero, right?